src/main/java/com/nirmata/workflow/serialization/JsonSerializer.java

```diff
@@ -318,7 +319,8 @@ static StartedTask getStartedTask(JsonNode node)
         return new StartedTask
         (
             node.get("instanceName").asText(),
-            LocalDateTime.parse(node.get("startDateUtc").asText(), DateTimeFormatter.ISO_DATE_TIME)
+            LocalDateTime.parse(node.get("startDateUtc").asText(), DateTimeFormatter.ISO_DATE_TIME),
+            node.get("progress").asInt()
```

Comment:
As progress is a new field, I think it's wise to check for it's presence first and use a default if not there. Otherwise, upgrading an existing workflow will fail.